polymorphic/admin/inlines.py

```diff
@@ -53,7 +53,8 @@ class PolymorphicInlineModelAdmin(InlineModelAdmin):
 
     #: Inlines for all model sub types that can be displayed in this inline.
     #: Each row is a :class:`PolymorphicInlineModelAdmin.Child`
-    child_inlines = ()
+    child_inlines = None
+    exclude_childs = None
```

Comment:
```suggestion
    exclude_children = None
```

This also needs a doc comment to go along with it.

Reply:
Why do you propose the change of `childs` for `children`? `childs` is used everywhere else in the source code...

Reply:
Where can you find `childs`? I (or `git grep`) certainly can't:

```
~/b/django-polymorphic (master) $ git grep children | wc -l
      70
~/b/django-polymorphic (master) $ git grep childs | wc -l
       0
```

Reply:
I still think `children` would be the correct spelling btw. :)

Reply:
Changed, I think I got confused with the usage of `child` (single) everywhere. You are right, `childs` is not used in any other part of the code, and also `children` is the gramatically correct alternative, thank you for drawing attention on this :-)

Reply:
Also added the documentation.